src/pages/Build/Research.vue

```diff
@@ -0,0 +1,248 @@
+<template>
+	<www-page>
+		<developer-secondary-menu slot="secondary" />
+		<div class="row page-content">
+			<div class="small-12 columns">
+				<h3>2018</h3>
+				<div v-if="paper.year === 2018" v-for="paper in papers" :key="paper">
```

Comment:
@go-bears Please update your v-for and :key attribtutes to work as shown below. Assigning the "paper" to key may technically work but key really just needs a unique flat attribute rather than the entire object.

`v-for="(paper, index) in papers" :key="index"`

Otherwise, everything looks great!

Reply:
thanks for the feed back & tip about the key! @BoulderBrains helped me a lot in figuring out the first draft of this loop.